### PR TITLE
Add reference model

### PR DESCRIPTION
**Pull request by elyousfi5**

### CODE REVIEW

File: src/ostorlab/serve_app/types.py [hunk 1]
```diff
@@ -52,9 +61,28 @@ class Meta:
...
         )
 
+    def resolve_references(
+        self: models.Vulnerability, info: graphql_base.ResolveInfo
+    ) -> List[OxoReferenceType]:
+        """Resolve references query.
+
+        Args:
+            self (models.Vulnerability): The vulnerability object.
+            info (graphql_base.ResolveInfo): GraphQL resolve info.
```
Comment (oussamaessaji) [inline src/ostorlab/serve_app/types.py:73]: no need for types

File: src/ostorlab/serve_app/types.py [hunk 2]
```diff
@@ -52,9 +61,28 @@ class Meta:
...
+        self: models.Vulnerability, info: graphql_base.ResolveInfo
+    ) -> List[OxoReferenceType]:
+        """Resolve references query.
+
+        Args:
+            self: The vulnerability object.
+            info: GraphQL resolve info.
+
+        Returns:
+            List[OxoReferenceType]: List of references.
```
Comment (benyissa) [inline src/ostorlab/serve_app/types.py:76]:
```suggestion
            List of references.
```

File: src/ostorlab/runtimes/local/models/models.py [hunk 3]
```diff
@@ -302,6 +309,35 @@ def create(key: str, value: str, scan_id: int):
             return scan_status
 
 
+class Reference(Base):
+    """The Reference model"""
+
+    __tablename__ = "reference"
+    id = sqlalchemy.Column(sqlalchemy.Integer, primary_key=True)
+    title = sqlalchemy.Column(sqlalchemy.String(255))
+    url = sqlalchemy.Column(sqlalchemy.String(255))
```
Comment (amine3) [inline src/ostorlab/runtimes/local/models/models.py:318]: Is 255 enough for the url?

**Review by amine3 — CHANGES_REQUESTED** [review]

File: src/ostorlab/runtimes/local/models/models.py (hunk 3 above)
Comment (elyousfi5) [reply src/ostorlab/runtimes/local/models/models.py]: @amine3 i updated to 4096

**Review by amine3 — APPROVED** [review]

**Pull request merged**